Automattic/jetpack
# Replace Akismet plan with feature

#### Changes proposed in this Pull Request:

* No functional changes, everything should work as before
* Moves from plan checks to feature checks to check if Akismet features is available.

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Go to /wp-admin/admin.php?page=jetpack#/dashboard on your Jetpack test site.
* Make sure the `Anti-spam` box looks something like below:
<img width="500" alt="Screenshot 2022-05-03 at 10 16 20 PM" src="https://user-images.githubusercontent.com/86406124/166500622-9db887cf-0f65-4805-9b4a-319e41277389.png">

* Click on the Upgrade button and buy the `Jetpack Anti-Spam` plan
* Complete the purchase and navigate back to /wp-admin/admin.php?page=jetpack#/dashboard again
* Now make sure the `Anti-spam` box looks something like below:
<img width="506" alt="Screenshot 2022-05-03 at 10 19 53 PM" src="https://user-images.githubusercontent.com/86406124/166501118-489b792d-0d2e-4350-820b-9ea9403a398b.png">

* Make sure to test without these changes, everything should work as before
